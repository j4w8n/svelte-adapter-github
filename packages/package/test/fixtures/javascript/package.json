{
	"name": "javascript",
	"private": true,
	"version": "1.0.0",
	"description": "standard javascript package"
}
