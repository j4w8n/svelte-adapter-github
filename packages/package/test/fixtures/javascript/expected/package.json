{
	"name": "javascript",
	"private": true,
	"version": "1.0.0",
	"description": "standard javascript package",
	"type": "module",
	"exports": {
		"./package.json": "./package.json",
		"./internal": "./internal/index.js",
		"./internal/Test.svelte": "./internal/Test.svelte",
		"./Test.svelte": "./Test.svelte",
		"./Test2.svelte": "./Test2.svelte",
		"./utils": "./utils.js",
		".": "./index.js"
	},
	"svelte": "./index.js"
}